packages/flutter_test/lib/src/controller.dart

```diff
@@ -72,9 +84,6 @@ class SemanticsController {
   /// if no semantics are found or are not enabled.
   SemanticsNode find(Finder finder) {
     TestAsyncUtils.guardSync();
-    if (_binding.pipelineOwner.semanticsOwner == null) {
-      throw StateError('Semantics are not enabled.');
-    }
```

Comment:
How come we can remove this?

Reply:
This is handled by the check in the `WidgetController.semantics` property (which protects all of the methods in this class). I forgot to remove it from this method when I initially added the check in the `semantics` property.